actions/lib/formatters.py

```diff
@@ -64,6 +63,20 @@ def to_issue_dict(issue, include_comments=False, include_attachments=False,
             item = to_attachment_dict(attachment)
             result['attachments'].append(item)
 
+    if include_components:
+        result['components'] = []
+
+        for component in issue.fields.components:
+            item = to_component_dict(component)
+            result['components'].append(item)
+
+    if include_subtasks:
+        result['subtasks'] = []
+
+        for subtask in issue.fields.subtasks:
+            item = to_subtask_dict(subtask)
+            result['subtasks'].append(item)
+
```

Comment:
This could be simplified using list comprehensions to the following:
```suggestion
    if include_components:
        result['components'] = [to_component_dict(c) for c in issue.fields.components]

    if include_subtasks:
        result['subtasks'] = [to_subtask_dict(s) for s in issue.fields.subtasks]
```